components/event-bus/cmd/event-bus-publish-knative/validators/publish_validator.go

```diff
@@ -27,6 +27,11 @@ func ValidateRequest(r *http.Request) (*api.PublishRequest, *api.Error) {
 	_ = r.Body.Close()
 	if err != nil {
 		log.Printf("failed to read request body: %v", err)
+
+		if err.Error() == "http: request body too large" {
```

Comment:
Could you please put "http: request body too large" in a const, so that if Golang changes the implementation of  MaxBytesReader: https://golang.org/src/net/http/request.go
we should easier react.